### PR TITLE
Get kitsu login to fill username burnin

### PR DESCRIPTION
**Pull request by friquette**
## Brief description
Get kitsu login to fill username burnin. when session username is not right.

## Description
Get kitsu login to fill username burnin when session username is not the right username (like multiple artists working on the same session).

### CODE REVIEW

**Comment by iLLiCiTiT** [comment]
This is not how we add addon specific data to burnins, it's not job of ExtractBurnin to know that. To achive that you can fill `"customData"` key on instance in kitsu collector to pass the data to burnins. In burnins you can then use `{custom["kitsuUsername"]}` for instance.

**Comment by friquette** [comment]
Thanks for the tips ! When you talk about kitsu collector, do you mean the collect_kitsu_entities file ?

**Comment by iLLiCiTiT** [comment]
I would recommend special collector just for that, so it can have it's own settings.

File: openpype/modules/kitsu/plugins/publish/collect_kitsu_username.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+
+class CollectKitsuUsername(pyblish.api.ContextPlugin):
+    """Collect Kitsu username from the kitsu login"""
+
+    order = pyblish.api.CollectorOrder + 0.499
+    label = "Kitsu username"
+
+    def process(self, context):
+        kitsu_login = os.environ['KITSU_LOGIN']
```
Comment (iLLiCiTiT) [inline openpype/modules/kitsu/plugins/publish/collect_kitsu_username.py:15]: No sure if `KITSU_LOGIN` env is always filled?
```suggestion
        kitsu_login = os.environ.get('KITSU_LOGIN')
```

**Review by iLLiCiTiT — APPROVED** [review]

**Pull request merged**